consent-ui/src/app/ais/entry-page/initiation/consent-initiate/consent-initiate.component.spec.ts

```diff
@@ -1,16 +1,26 @@
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-import {ConsentInitiateComponent} from './consent-initiate.component';
+import { ConsentInitiateComponent } from './consent-initiate.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { ActivatedRoute, convertToParamMap } from '@angular/router';
 
 describe('ConsentInitiateComponent', () => {
   let component: ConsentInitiateComponent;
   let fixture: ComponentFixture<ConsentInitiateComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ConsentInitiateComponent ]
-    })
-    .compileComponents();
+      declarations: [ConsentInitiateComponent],
+      imports: [ReactiveFormsModule, HttpClientTestingModule, RouterTestingModule],
+      providers: [
+        {
+          provide: ActivatedRoute,
+          useValue: { snapshot: { params: { authId: 'AUTH-ID' }, queryParams: { redirectCode: 'REDIRECT-CODE' } } }
```

Comment:
Suggest to replace multiple occurrences of 'REDIRECT-CODE' with some constant

Reply:
done